backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentDatabaseHandler.java

```diff
@@ -700,8 +700,12 @@ private void removeReleaseId(String releaseId, String componentId) throws SW360E
         return getLinkedReleases(relations, releaseMap, new ArrayDeque<>());
     }
 
+    public List<Release> getAllReleases() {
```

Comment:
There are two more calls of `releaseRepository.getAll()` in this class, which should be replaced with calls of this method

Reply:
DONE